2019-03-09 09:13:58: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:13:58: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:13:58: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:13:58: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:48:47: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:56:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:56:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 09:56:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 10:02:33: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 10:29:22: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 10:52:55: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 10:52:55: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 10:52:55: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 10:52:55: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:11:05: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:30:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:30:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:30:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:30:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:30:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:30:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 11:43:45: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:03: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:50:04: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:58:38: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:58:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:58:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:58:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:58:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 11:58:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 12:00:17: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:49:30: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:34: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:35: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 15:54:35: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:14:17: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:16:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:21:16: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:44:57: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:45:23: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:45:23: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:45:23: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:45:23: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 16:48:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:00:09: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:01:15: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:01:53: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:10: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:10: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:09:11: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:24:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:24:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:24:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-03-09 17:24:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
